.github/scripts/archive-workflows.sh

```
@@ -0,0 +1,39 @@
+  "core-deploy.yml"
+  "core-security.yml"
+  "README.md"
+)
+
+# Move to workflows directory
+cd ../.github/workflows
+
+# Process each workflow file
+for file in *.yml; do
+  # Skip core workflows
+  if [[ " ${CORE_WORKFLOWS[@]} " =~ " ${file} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)